io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl

```diff
@@ -0,0 +1,417 @@
+   samm:see <https://www.mdsystem.com/imdsnt/startpage/index.jsp> ;
+   samm:dataType xsd:string .
+
+:MaterialNameStandardized a samm:Characteristic ;
+   samm:preferredName "Material Name Standardized"@en ;
+   samm:description "Material name according to standard."@en ;
+   samm:dataType :MaterialNameStandardizedEntity .
+
+:UnitOfMeasure a samm:Characteristic ;
+   samm:preferredName "Unit of Measure"@en ;
+   samm:description "Attribute to define the measure and actual mass of the material."@en ;
+   samm:dataType :UnitOfMeasureEntity .
+
+:BioBased a samm:Characteristic ;
+   samm:preferredName "Bio Based"@en ;
+   samm:description "Bio-based materials are materials of biological origin excluding material embedded in geological formations and/or fossilised."@en ;
+   samm:dataType :BioBasedEntity .
+
+:Inorganic a samm:Characteristic ;
+   samm:preferredName "Inorganic"@en ;
+   samm:description "Material based on inorganic matter or fossil fuels."@en ;
+   samm:dataType :InorganicEntity .
+
+:Certificate a samm-c:Set ;
+   samm:preferredName "Certificate"@en ;
+   samm:description "If available, the name of a certificate verifying the recycling content can be included."@en ;
+   samm:dataType :CertificateEntity .
+
+:MaterialNameStandardizedEntity a samm:Entity ;
+   samm:preferredName "Material NameStandardized Entity"@en ;
+   samm:description "Material name according to standard."@en ;
+   samm:properties ( :referencedStandard :referencedStandardID :materialNameStandardizedValue ) .
+
+:UnitOfMeasureEntity a samm:Entity ;
+   samm:preferredName "Unit of Measure Entity"@en ;
+   samm:description "Attribute to define the measure and actual mass of the material."@en ;
+   samm:properties ( :unitOfMeasureKey :unitOfMeasureValue ) .
+
+:BioBasedEntity a samm:Entity ;
+   samm:preferredName "Bio Based Entity"@en ;
+   samm:description "Bio-based materials are materials of biological origin excluding material embedded in geological formations and/or fossilised."@en ;
+   samm:properties ( :bioBasedClass :secondaryBioBased :primaryBioBased ) .
+
+:InorganicEntity a samm:Entity ;
+   samm:preferredName "Inorganic Entity"@en ;
+   samm:description "Material based on inorganic matter or fossil fuels."@en ;
+   samm:properties ( :secondaryInorganic :primaryInorganic ) .
+
+:CertificateEntity a samm:Entity ;
+   samm:preferredName "Certificate Entity"@en ;
+   samm:description "If available, the name of a certificate verifying the recycling content can be included."@en ;
+   samm:properties ( :certificateName :certificateLink ) .
+
+:referencedStandard a samm:Property ;
+   samm:preferredName "referenced Standard"@en ;
+   samm:description "Norm authority who governs norm."@en ;
+   samm:characteristic :ReferencedStandard ;
+   samm:exampleValue "ISO" .
+
+:referencedStandardID a samm:Property ;
+   samm:preferredName "referenced Standard ID"@en ;
+   samm:description "Norm code of related naming norm."@en ;
```

Comment:
don't get the description. Maybe something like
Id of standard used by standardization body?